src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -347,6 +350,12 @@ public void DeleteDatabase(string dbName, DeletionInProgressStatus deletionInPro
                     {
                         configuration = CreateDatabaseConfiguration(dbName, ignoreDisabledDatabase: true, ignoreBeenDeleted: true, ignoreNotRelevant: true,
                             databaseRecord: record);
+
+                        CheckDatabasePathsIntersection(dbName, configuration);
+                    }
+                    catch (InvalidOperationException)
+                    {
+                        throw;
                     }
```

Comment:
Avoid the `catch` here put the `CheckDatabasePathsIntersection` after the try,

Reply:
Done.